recipes/apriltag/all/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project(cmake_wrapper)
+cmake_minimum_required(VERSION 3.1)
+set (CMAKE_C_STANDARD 11)
```

Comment:
do not use `set` for global configuration, it's a bad practice in CMake, prefer set_property (cmake 2.8) or set_target_properties

Reply:
I know but in order to use set_property I need to specify a target and the apriltag target is not defined in the CMake wrapper. I think the only way to do make it work would be to create a patch for the original CMakeLists. Do you think there's a better way to do it?
By the way, the yaml-cpp CMake wrapper also uses this approach.

Reply:
@ericriff take a look at  ericriff#1 there is a treatment for it there

Reply:
```suggestion
```

You no longer need it.